.ci/configure_crio_for_kata.sh

```diff
@@ -34,6 +34,10 @@ default_runtime = "kata"
 runtime_path = "/usr/local/bin/kata-runtime"
 runtime_root = "/run/vc"
 runtime_type = "oci"
+[crio.runtime.runtimes.containerd-shim-kata-v2]
```

Comment:
Hi @fidencio , this change looks good to me as is. But I'd like to know what you think about adding a logic to set the default runtime as well, now that cri-o will be configured with 2 entries. IMHO we should do it.

Also I noticed that in `.ci/lib.sh` it is exported `KATA_RUNTIME=${KATA-RUNTIME:-kata-runtime}` but it seems not being used. On scripts that need to switch between `kata-runtime` and `containerd-shim-kata-v2`, it is being use `RUNTIME=${RUNTIME:-kata-runtime}` instead (e.g `./integration/kubernetes/init.sh`) . And it is an inconsistency, maybe we should improve it (in another PR). What do you think @GabyCT ?

Reply:
> Hi @fidencio , this change looks good to me as is. But I'd like to know what you think about adding a logic to set the default runtime as well, now that cri-o will be configured with 2 entries. IMHO we should do it.

@wainersm, we can simply force a default runtime using the `CONTAINER_DEFAULT_RUNTIME` environment variable, that may be the most reliable way to do so.

Would that be enough or do you have something different in mind?

Reply:
hi @fidencio, is the `CONTAINER_DEFAULT_RUNTIME` being used somewhere else or already exported in the environment? Asking because if this is a new variable you proposing, maybe we should stick with `KATA_RUNTIME` instead.